scripts/clickhouse_operator_sync.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+#
+# This tool fetch and format 'Altinity/clickhouse-operator'
+# k8s resource definitions into our chart.
+#
+# Why do we need this? The 'clickhouse-operator' doesn't expose a Helm
+# package so we need to collect and bundle the resources by our own.
+#
+
+set -o errexit
+set -o nounset
+
+CURRENT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+CHART_PATH_RAW="${CURRENT_DIR}/../charts/posthog"
+CHART_PATH=$(cd "$CHART_PATH_RAW" 2> /dev/null && pwd -P)
+TMP_FOLDER="$(mktemp -d)"
+trap 'rm -rf -- "$TMP_FOLDER"' EXIT
+
+OPERATOR_NAMESPACE="posthog"
+CLICKHOUSE_OPERATOR_TAG="0.16.1"
+URL="https://raw.githubusercontent.com/Altinity/clickhouse-operator/${CLICKHOUSE_OPERATOR_TAG}/deploy/operator/clickhouse-operator-install-template.yaml"
+
+#
+# Download the 'altinity/clickhouse-operator' definition and save it as temporary file.
+#
+# see: https://github.com/Altinity/clickhouse-operator/blob/master/docs/quick_start.md#in-case-you-can-not-run-scripts-from-internet-in-your-protected-environment
+#
+OPERATOR_NAMESPACE="${OPERATOR_NAMESPACE:-test-clickhouse-operator}"
+METRICS_EXPORTER_NAMESPACE="${OPERATOR_NAMESPACE}"
+OPERATOR_IMAGE="${OPERATOR_IMAGE:-altinity/clickhouse-operator:latest}"
+METRICS_EXPORTER_IMAGE="${METRICS_EXPORTER_IMAGE:-altinity/metrics-exporter:latest}"
+
+curl -s "${URL}" | \
+    OPERATOR_IMAGE="${OPERATOR_IMAGE}" \
+    OPERATOR_NAMESPACE="${OPERATOR_NAMESPACE}" \
+    METRICS_EXPORTER_IMAGE="${METRICS_EXPORTER_IMAGE}" \
+    METRICS_EXPORTER_NAMESPACE="${METRICS_EXPORTER_NAMESPACE}" \
+    envsubst > "$TMP_FOLDER/clickhouse-operator.yaml"
+
+#
+# Use 'altinity/clickhouse-operator' definition file we fetched and parsed and slice it
+# in different files, based on the resource kind
+#
+go install github.com/patrickdappollonio/kubectl-slice@v1.1.0
```

Comment:
Is it expected elsewhere that `go` will be available?

Reply:
Also, what's the reason we need to split the files?

Reply:
> Is it expected elsewhere that go will be available?

no and it's a fair point, I should probably add it as requirement

> Also, what's the reason we need to split the file?

To don't end up with a file of thousands line 😅. I prefer to have different files also for more isolated unit testing, etc..